src/sidebar/components/Annotation/AnnotationEditor.tsx

```diff
@@ -178,6 +179,28 @@ function AnnotationEditor({
   const mentionsEnabled = store.isFeatureEnabled('at_mentions');
   const usersWhoAnnotated = store.usersWhoAnnotated();
   const focusedGroupMembers = store.getFocusedGroupMembers();
+  const usersForMentions = useMemo(() => {
```

Comment:
```suggestion
  const usersForMentions = useMemo(() => {
```

I think it would be helpful for readability to give this variable an explicit type, as it isn't just a simple array.